ui.apps/src/main/content/jcr_root/apps/core/cif/components/commerce/featuredcategorylist/v1/featuredcategorylist/_cq_dialog/.content.xml

```diff
@@ -30,6 +30,10 @@
                             jcr:primaryType="nt:unstructured"
                             sling:resourceType="core/wcm/components/commons/datasources/allowedheadingelements/v1"/>
                     </titleType>
+                    <categoryIdType jcr:primaryType="nt:unstructured" 
+                        sling:resourceType="granite/ui/components/coral/foundation/form/hidden" 
+                        name="./categoryIdType"
+                        value="id"/>
```

Comment:
How is this value changed when the customer migrates to using UIDs?

Reply:
I changed it to match the `categoryId` - `categoryIdType` on each item.
But the way the Category Picker works right now is via `selection-id` property with possible values : `id`, `idAndSlug` and `uid`.

For the featuredCategory component, that value is passed down from https://github.com/adobe/aem-core-cif-components/blob/63f616040560215c8cde3da8562ff480218a6a31/ui.apps/src/main/content/jcr_root/apps/core/cif/components/commerce/featuredcategorylist/v1/featuredcategorylist/_cq_dialog/.content.xml#L52

So, your question still remains